packages/schema/src/config/build.ts

```diff
@@ -38,6 +39,20 @@ export default defineUntypedSchema({
     },
   },
 
+  /**
+   * Log level when building logs.
+   *
+   * Defaults to 'silent' when running in CI or when a TTY is not available.
+   * This option is then used as 'silent' in Vite and 'none' in Webpack
+   *
+   * @type {'silent' | 'info'}
+   */
+  logLevel: {
+    $resolve: (val) => {
+      return val ?? (isMinimal ? 'silent' : null)
```

Comment:
`isMinimal` includes when there is no TTY (basic CLIs) we still can log in there. We should use `isTest` only.

Reply:
Changed in [a1f4f5c](https://github.com/nuxt/nuxt/pull/19369/commits/a1f4f5c24d5d6b6d9c04b258fcbd7018fad9e67c)